.evergreen.yml

```diff
@@ -56,43 +140,73 @@ functions:
       binary: npm
       args: [run, coverage]
 
-  attach-results:
+  run-cypress-tests:
+    command: shell.exec
+    params:
+      working_dir: spruce
+      script: |
+        Xvfb :1 -screen 0 1280x800x24 &
+        export DISPLAY=:1
+        npx cypress run --record --key ${cypress_record_key}
```

Comment:
Is there a way to run this in non-interactive mode? Looking at the task for this PR, it's hanging and will probably time out, and I think it's because it's waiting for input

Reply:
I think interactive mode is set with `cypress open`. The commit this code change belonged to had reasonable output and passed tests: https://evergreen.mongodb.com/task/spruce_ubuntu1604_e2e_test_patch_10233104f2a3b4870baf72fba9a3632790dbae36_5e28a536562343334d346dcb_20_01_22_19_40_41

Reply:
Yeah it's not in interactive mode. The hanging issue must be caused by something else.

Reply:
I think it had to do with the ubuntu 18 workaround because everything works fine on ubuntu 16. Seems like ubuntu 18 will cause problems until the fix that will be merged in Cypress v3.8.3. Until then we should use ubuntu 16